main.go

```diff
@@ -0,0 +1,335 @@
+package main
+
+import (
+	"net/http"
+	"html/template"
+	"github.com/gorilla/mux"
+	"os"
+	"log"
+	"strings"
+	"bufio"
+	"database/sql"
+	//"github.com/coopernurse/gorp"
+	//"encoding/json"
+	_"github.com/go-sql-driver/mysql"
+  	"github.com/goincremental/negroni-sessions"
+	//"github.com/goincremental/negroni-sessions/cookiestore"
+	"github.com/urfave/negroni"
+	"golang.org/x/crypto/bcrypt"
+	"github.com/goincremental/negroni-sessions/cookiestore"
+	"fmt"
+//	"os/user"
+	"strconv"
+	//"golang.org/x/tools/go/gcimporter15/testdata"
+	"encoding/json"
+)
+type User struct {
+	 Username string `db:"username"`
+	 Password []byte `db:"userpassword"`
+}
+type contact_info struct{
+	Name   		string   `db:"name"`
+	Number 		string   `db:"number"`
+	Email  		string   `db:"email"`
+	Nationality 	string   `db:"nationality"`
+	Address 	string   `db:"address"`
+	Username  	string   `db:"username"`
+	ID 		int   `db:"id"`
+}
+type LoginPage struct {
+	Error string
+}
+type page struct{
+	Contacts []contact_info
+	Numbers []Telephone
+}
+type Telephone struct {
+	Contact_id int `db:"contactid"`
+	Number string  	`db:"number"`
+	Num_id int	`db:"numid"`
+
+}
+var muxer = mux.NewRouter()
+func main(){
+	initDb()
+
+	var contact contact_info
+	var userAccount User
+	var telephone Telephone
+
+	muxer.HandleFunc("/viewNumbers/{id}", telephone.ViewTelephonesHandler).Methods("GET")
```

Comment:
- URLs should be all small letters
- Although making contact, userAccount, telephone as receivers is a good, but it has no use here.